pkg/crypto/tinkcrypto/primitive/composite/ecdhes/ecdhes_decrypt_factory.go

```diff
@@ -65,6 +65,7 @@ func newDecryptPrimitiveSet(ps *primitiveset.PrimitiveSet) (*decryptPrimitiveSet
 // Decrypt decrypts the given ciphertext and authenticates it with the given
 // additional authenticated data. It returns the corresponding plaintext if the
 // ciphertext is authenticated.
+// nolint: nestif
```

Comment:
Comment would be helpful.

Reply:
@soluchok I think this nolint comment should be removed

Reply:
you need to apply the same refactoring logic you did as in pkg/crypto/tinkcrypto/primitive/composite/ecdh1pu/ecdh1pu_decrypt_factory.go 

the same code is found here too, then you can remove `// nolint: nestif`

it does not make sense to refactor ecdh1pu and put nolint in ecdhes.

Reply:
can you see this comment @soluchok @troyronda  ?

Reply:
refactoring seems good.. only the `//nolint` comment needs to be removed here